src/main/java/core/basesyntax/RandomFigure.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomFigure {
+    private static final String[] figures = new String[]
+            {"Square", "Rectangle", "RightTriangle", "Circle", "IsoscelesTrapezoid"};
+
+    public Figure getFigure() {
+
+        int colorIndex = new Random().nextInt(Color.values().length);
+        Color randomColor = Color.values()[colorIndex];
+
+        switch (figures[new Random().nextInt(figures.length)]) {
+            case "Square":
+                double side = new Random().nextInt(15);
+                return new Square(side, randomColor.toString());
+            case "Rectangle":
+                double sideA = new Random().nextInt(15);
+                double sideB = new Random().nextInt(20);
+                return new Rectangle(sideA, sideB, randomColor.toString());
+            case "RightTriangle":
+                double sideT = new Random().nextInt(15);
+                double base = new Random().nextInt(20);
+                return new RightTriangle(sideT, base, randomColor.toString());
+            case "Circle":
+                double radius = new Random().nextInt(20);
+                return new Circle(radius, randomColor.toString());
+            case "IsoscelesTrapezoid":
+                double topSide = new Random().nextInt(20);
```

Comment:
For future, if methods grows bigger, you may split it into smaller ones to make code more readable

Reply:
Thank you, from now on I will do like that